src/core/dbus-service.c

```diff
@@ -532,6 +562,43 @@ static int bus_service_set_transient_property(
         if (streq(name, "StandardErrorFileDescriptor"))
                 return bus_set_transient_std_fd(u, name, &s->stderr_fd, &s->exec_context.stdio_as_fds, message, flags, error);
 
+        if (streq(name, "OpenFile")) {
+                char *path = NULL;
+                char *fdname = NULL;
+                uint64_t options;
```

Comment:
This should be named `flags`

Reply:
There is already a `flags` variable in the outer scope.

Reply:
Let's call it `offlags` then